src/utils/env.ts

```diff
@@ -1,15 +1,10 @@
-// Use relative import to resolve during nuxt runtime
-import stringToBoolean from './string-to-boolean'
-
 /**
  * Default environment variables are set on this key. Defaults are fallbacks to existing env vars.
  * All boolean values should be designed to be false by default.
  */
-export const env = {
+export const env: Record<string, string> = {
   apiUrl: process.env.API_URL ?? 'https://api.openverse.engineering/v1/',
   filterStorageKey: 'openverse-filter-visibility',
   notificationStorageKey: 'openverse-show-notification',
-  enableInternalAnalytics: stringToBoolean(
-    process.env.ENABLE_INTERNAL_ANALYTICS
-  ),
+  enableInternalAnalytics: process.env.ENABLE_INTERNAL_ANALYTICS ?? 'false',
 }
```

Comment:
Let's use `as const` here: it's perfect for this kind of thing. `Record<string, string>` is actually even worse than what TypeScript infers as it swallows all the key names. `as const` is the best though as it will turn constant strings into constant string types instead of just `string`. Changing `enableInternalAnalytics` to a `string` boolean representation instead of just a plain boolean also strikes me as a step backwards, what was the reason for that?

Reply:
Makes sense! I wonder what a good way to enforce that would be :thinking: I guess `Record<string, string>` would be it but having the keys swallowed is a bit of a bummer :confused:

Reply:
We wouldn't need to enforce this, `tsc` will raise an error if we use anything other than a string as the value. Also using `as const` with it works.